app/controllers/admin/graphs_controller.rb

```diff
@@ -35,7 +35,10 @@ def variable
                         .count}
                     ]
       elsif @kind == "recoveries"
-        chart_data = StolenRecord.where(date_recovered: @start_at..@end_at)
+        recoveries = StolenRecord.unscoped
+        chart_data = recoveries.where(date_recovered: @start_at..@end_at)
+                         .group_by_period(@group_period, :date_recovered, time_zone: @timezone)
+                         .count
```

Comment:
I checked all the other graphs, they seem to working well enough, this is the only one I was getting an error on because... I did a really bad job. Seems to be working now.